provisionopenvpn_policy.tf

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows all of the permissions necessary
+# to provision OpenVPN in the Shared Services account.
+# ------------------------------------------------------------------------------
+
+data "aws_iam_policy_document" "provisionopenvpn_policy_doc" {
+  # This policy needs to be more restricted, obviously
```

Comment:
🎗Reminder to create an issue to go back and restrict this policy (if this PR is merged and this policy is still wide open).